### ISSUE TITLE
Add DPO Contact Field 

### ISSUE THREAD

**Issue by smartopian** [KantaraInitiative/CISWG#101]
In the GDPR - there is a point for for a DPO contact field, - which different from the On behalf contact, could, be a field for a privacy officer, or point of contact/end point for privacy and consent related requests. 

This issue was actually something we accounted for prior to the GDPR in the MVCR v.0.9 where we did have a privacy contact field.  But this was confused with the Data Controller Contact info and dropped.  

Technically, this field would have been usable for this GDPR Article Clause,13-1(b)  with the guidance that this is a designated privacy contact. 

With a string format for contact information. 

In terms of implementations, I know that the on-behalf field and the privacy contact field have been used together to designate a trusted 3rd party as proxy. 

As a result I recommend bringing back the privacy point of contact field, with a GDPR friendly reference that this field can be used for DPO where applicable in the GDPR. 
Where I believe law requires a DPO to be listed for a company of a certain size. 

Here is the GDPR Reference -  Article 13 - 1 (b) - 

**Article 13**
Information to be provided where personal data are collected from the data subject
1. Where personal data relating to a data subject are collected from the data subject, the controller shall, at the time when personal data are obtained, provide the data subject with all of the following information:
(a) the identity and the contact details of the controller and, where applicable, of the controller's representative;
 ** (b) **the contact details of the data protection officer, where applicable;**
(c) the purposes of the processing for which the personal data are intended as well as the legal basis for the processing;

**Comment by PrivacyCDN** [comment]
I'm of a mind to let simple prevail. If a person has a question or concern about a CR they should have a single point of contact. GDPR implementations would probably put in the DPO contact info, but some organizations could choose a customer contact and depend on their internal routing to get it to the right office.

Thanks, John
4giv spellin errurz from mobile devize
________________________________

**Comment by smartopian** [comment]
I agree with you John, instead of making this specifically a DPO field or even a Privacy field, having a direct contact, or first contact field for all privacy complaints, which can be define by the data controller I think is itself simple and easy.   Guidance: We can add this as a May field and not required because a) the data controller is the direct contract, then its not used.  If its require for compliance or their is an alternative contact (aKA, agent, service provider, whatever) this can then be listed here.   (IMO) this field is needed to be added back in, but, the lesson learned is,  it doesn't need to be optimised.
